package.json

```diff
@@ -1,64 +1,67 @@
 {
-  "name": "monster-hunter-now-events",
-  "description": "A tool that auto-generates calendar events for Monster Hunter Now by scraping web news articles, processing them with AI, and creating a convenient calendar subscription.",
-  "version": "1.1.2",
-  "author": "Sean Kennedy <sean@codeserious.com>",
-  "bugs": {
-    "url": "https://github.com/seriouslysean/monster-hunter-now-events/issues"
-  },
-  "engines": {
-    "node": ">=18.0.0",
-    "npm": ">=9.6.0"
-  },
-  "dependencies": {
-    "axios": "^1.5.1",
-    "commander": "^11.0.0",
-    "date-fns": "^2.30.0",
-    "node-html-parser": "^6.1.10",
-    "openai": "^4.11.0",
-    "rollbar": "^2.26.2"
-  },
-  "devDependencies": {
-    "dotenv": "^16.3.1",
-    "eslint": "^8.50.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-import": "^2.28.1",
-    "eslint-plugin-prettier": "^5.0.0",
-    "husky": "^8.0.3",
-    "prettier": "^3.0.3"
-  },
-  "homepage": "https://github.com/seriouslysean/monster-hunter-now-events#readme",
-  "keywords": [
-    "calendar",
-    "hacktoberfest",
-    "ics",
-    "javascript",
-    "monster-hunter",
-    "monster-hunter-now",
-    "nodejs",
-    "openai",
-    "web-scraping"
-  ],
-  "license": "MIT",
-  "main": "index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/seriouslysean/monster-hunter-now-events.git"
-  },
-  "scripts": {
-    "_lint:format": "prettier src --write",
-    "generate:events-json": "node -r dotenv/config ./src/generate-events-json.js",
-    "generate:events-ics": "node -r dotenv/config index.js",
-    "lint:fix": "prettier --write --list-different src && npm run lint:js --fix",
-    "lint:js": "eslint --ext .js --ignore-path .gitignore src/*",
-    "lint:prettier": "prettier --check src",
-    "lint": "npm run lint:js && npm run lint:prettier",
-    "start": "npm run fetch:all-articles && npm run generate:events-json && npm run generate:events-ics",
-    "fetch:all-articles": "node -r dotenv/config ./tools/fetch-all-articles.js",
-    "fetch:article": "node -r dotenv/config ./tools/fetch-article.js",
-    "test:logger": "node -r dotenv/config ./tools/test-logger.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "type": "module"
```

Comment:
It seems like all the spacing changed in this file, was that intentional?

Reply:
No, not intentional at all! Though I suspect this happened because of my VSCode ESLint extension. I'll try rectifying this.